README.md

```diff
@@ -5,46 +5,196 @@
 ![Azure DevOps builds (branch)](https://img.shields.io/azure-devops/build/openvinoci/b2bab62f-ab2f-4871-a538-86ea1be7d20f/13?label=Public%20CI)
 [![PyPI Downloads](https://pepy.tech/badge/openvino)](https://pepy.tech/project/openvino)
 
-This toolkit allows developers to deploy pre-trained deep learning models
-through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
+## Contents:
+
+ - [What is OpenVINO?](#what-is-openvino-toolkit)
+    - [Components](#components)
+ - [Supported Hardware matrix](#supported-hardware-matrix)
+ - [License](#license)
+ - [Documentation](#documentation)
+ - [Tutorials](#tutorials)
+ - [Products which use OpenVINO](#products-which-use-openvino)
+ - [System requirements](#system-requirements)
+ - [How to build](#how-to-build)
+ - [How to contribute](#how-to-contribute)
+ - [Get a support](#get-a-support)
+ - [See also](#see-also)
+
+## What is OpenVINO toolkit?
+
+OpenVINO™ toolkit allows developers to deploy pre-trained deep learning models through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
 
 This open source version includes several components: namely [Model Optimizer], [OpenVINO™ Runtime], [Post-Training Optimization Tool], as well as CPU, GPU, MYRIAD, multi device and heterogeneous plugins to accelerate deep learning inferencing on Intel® CPUs and Intel® Processor Graphics.
 It supports pre-trained models from the [Open Model Zoo], along with 100+ open
 source and public models in popular formats such as TensorFlow, ONNX, PaddlePaddle, MXNet, Caffe, Kaldi.
 
-## Repository components
-* [OpenVINO™ Runtime]
-* [Model Optimizer]
-* [Post-Training Optimization Tool]
-* [Samples]
+### Components
+* [OpenVINO™ Runtime] - is a set of C++ libraries with C and Python bindings providing a common API to deliver inference solutions on the platform of your choice.
```

Comment:
When I click on the link, it says page not found. Perhaps it's because the link is not provided here?

Reply:
The link was changed